utils/navigator_msgs/srv/ObjectDBQuery.srv

```diff
@@ -0,0 +1,5 @@
+string name
+string cmd
```

Comment:
Could you add some comments into this service file about its usage? For example, what does cmd do?